.github/workflows/testing.yml

```diff
@@ -24,7 +24,6 @@ jobs:
           - '2.8'
           - '2.x-latest'
         python:
-          - '2.7'
```

Comment:
I guess we should also reflect it in `classifiers` and `python_requires` in `setup.py`.

Examples from `requests` for inspiration[^1]:

* https://github.com/psf/requests/blob/2d5517682b3b38547634d153cea43d48fbc8cdb5/setup.py#L107-L116
* https://github.com/psf/requests/blob/2d5517682b3b38547634d153cea43d48fbc8cdb5/setup.py#L96

[^1]: It supports Python 2 & 3, so maybe there is a better example. 

It also worth to look what the documentation suggests on the topic: https://packaging.python.org/en/latest/guides/dropping-older-python-versions/

Sorry, this thought just comes into my head.

Reply:
Moved the requirement to setup script. I think more detailed changes could be considered as a part of https://github.com/tarantool/tarantool-python/issues/212

Reply:
Okay.